app/components/osf-navbar/component.ts

```diff
@@ -49,12 +52,21 @@ export default class OsfNavbar extends Component {
     indexRoute: string = 'dashboard';
     showNavLinks: boolean = false;
 
-    osfApps = osfServices;
+    @computed('features')
+    get osfApps(this: OsfNavbar) {
+        if (this.features.isEnabled(config.featureFlags.routes.institutions)) {
+            return osfServices;
+        }
+        return osfServices.filter(e => e.name !== 'INSTITUTIONS');
```

Comment:
Instead of this, I think we should add a flag key to `osfServices` and, if defined, check the flag to see if it should be included. Something like:
```TypeScript
return osfServices.filter(service => !service.flag || this.features.isEnabled(service.flag));
```
see below for example in service-links